Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN pip install poetry==1.5.1
 
 # export requirements after copying req files
 COPY pyproject.toml poetry.lock ./
-RUN poetry export --without-hashes > requirements.txt
+RUN poetry export --with=database --without-hashes > requirements.txt
```

Comment:
Correct me if I'm wrong, but this shouldn't be needed since the dependency group isn't marked as optional

Reply:
Oh, I hadn't known that, my mistake 😂